nixos/modules/services/networking/tailscale.nix

```diff
@@ -0,0 +1,54 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.services.tailscale;
+in {
+  meta.maintainers = with maintainers; [ danderson mbaillie ];
+
+  options.services.tailscale = {
+    enable = mkEnableOption "Tailscale daemon";
+
+    package = mkOption {
+      type = types.package;
+      default = pkgs.tailscale;
+      defaultText = "pkgs.tailscale";
+      description = "The package providing Tailscale.";
+    };
+
+    port = mkOption {
+      type = types.port;
+      default = 41641;
+      description = "The port to listen on for tunnel traffic (0=autoselect).";
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.services.tailscale = {
+      description = "Tailscale daemon";
+      path = with pkgs; [ iproute iptables ];
```

Comment:
If these are required by the tailscale binary they should be included in the package with wrapProgram, so that people not using the module still have the appropriate dependencies.

Reply:
Can confirm that these are required by the `tailscaled` binary in the derivation. Unfortunately that does mean that the `tailscale` CLI (which is what most people on NixOS would use directly) will pull in iproute and iptables as well, but honestly those are probably already pulled in by other things anyway.

Reply:
I've now moved these to `wrapProgram` in the package itself.